### ISSUE TITLE
Provide configuration instead of hard-coding PROP_SAVE

### ISSUE THREAD

**Issue by prismofeverything** [braysia/CellTK#7]
There are some options that are provided at the beginning of `celltk/apply.py` under the value `PROP_SAVE` that sometimes need to be adjusted. It would be optimal if these could be provided in some kind of configuration file so that a user wouldn't have to edit the code to make these changes. Does CellTK already have a configuration file we could use?

**Comment by braysia** [comment]
I added celltk/_setting.py so that it can store a few hard-coded parameters.

It assumes users who install CellTK through pip does not need to modify these settings. It would be nicer if this file will be placed under a root directory in the future for the visibility.

**Comment by prismofeverything** [comment]
Right, there are a couple options for user provided configuration. One is to pass a config yaml in on the command line on invocation, like so: 

    > python celltk/command.py -c config.yaml -i data/testimages0/CFP/img* -f constant_thres -p THRES=2000 -o output/c1

This option could be present for all of the commands:

    > python celltk/caller.py -c config.yaml input_files/input_tests1.yml

The contents of this file could be parsed and then passed into the required part of the code. Here it would be `apply.py` but with this facility I'm sure additional configuration options would be added as time goes on.
